src/borg/archiver.py

```diff
@@ -1022,17 +1022,18 @@ def compare_content(path, item1, item2):
             if contents_changed(item1, item2):
                 if item1.get('deleted'):
                     return ('added {:>13}'.format(format_file_size(sum_chunk_size(item2))))
-                elif item2.get('deleted'):
+                if item2.get('deleted'):
                     return ('removed {:>11}'.format(format_file_size(sum_chunk_size(item1))))
-                else:
-                    chunk_ids1 = {c.id for c in item1.chunks}
-                    chunk_ids2 = {c.id for c in item2.chunks}
-                    added_ids = chunk_ids2 - chunk_ids1
-                    removed_ids = chunk_ids1 - chunk_ids2
-                    added = sum_chunk_size(item2, added_ids)
-                    removed = sum_chunk_size(item1, removed_ids)
-                    return ('{:>9} {:>9}'.format(format_file_size(added, precision=1, sign=True),
-                                                 format_file_size(-removed, precision=1, sign=True)))
+                if not can_compare_chunk_ids:
+                    return ('modified')
```

Comment:
https://github.com/borgbackup/borg/pull/2900/files#diff-48f2b9140a35ffd45a46ccfbe63931a6R1022
Basically this entire part is behind `if contents_changed(item1, item2):` which actually checks chunk contents if ids are not comparable.